SmellDetector/Detectors/RuleEngines/MetricCriteria.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using SmellDetector.Controllers;
+
+namespace SmellDetector.Detectors.RuleEngines
+{
+    public class MetricCriteria : Criteria
+    {
+        //This should be a single metric, so that we can compute the rule.
```

Comment:
Avoid comments - they are evil https://youtu.be/HskIvd9bpik

Reply:
I forgot to delete this comment. I wrote it so I don't forget what's the purpose for this field.